test/vm.js

```diff
@@ -1,74 +1,39 @@
-var Web3 = require("web3");
-var assert = require("assert");
-var Ganache = require(process.env.TEST_BUILD
-  ? "../build/ganache.core." + process.env.TEST_BUILD + ".js"
-  : "../index.js");
-var solc = require("solc");
-var fs = require("fs");
-
-var logger = {
-  log: function(message) {
-    // console.log(message);
-  }
-};
-
-var web3 = new Web3();
-web3.setProvider(
-  Ganache.provider({
-    /* blockTime: 100, */
-    logger: logger,
-    seed: "1337"
-  })
-);
+const assert = require("assert");
```

Comment:
Is this file supposed to be here?

Reply:
No. I'll restore from `develop` in the next commit.